archive/imo/imo1959_q1.lean

```diff
@@ -18,15 +18,16 @@ as saying the numerator and denominator are relatively prime.
 
 open nat
 
-namespace imo
+namespace imo1959_q1
 
 lemma calculation (n k : ℕ) (h1 : k ∣ 21 * n + 4) (h2 : k ∣ 14 * n + 3) : k ∣ 1 :=
 have h3 : k ∣ 2 * (21 * n + 4), from h1.mul_left 2,
 have h4 : k ∣ 3 * (14 * n + 3), from h2.mul_left 3,
 have h5 : 3 * (14 * n + 3) = 2 * (21 * n + 4) + 1, by ring,
 (nat.dvd_add_right h3).mp (h5 ▸ h4)
 
+@[nolint dup_namespace]
 theorem imo1959_q1 : ∀ n : ℕ, coprime (21 * n + 4) (14 * n + 3) :=
```

Comment:
Alternatively, why not move this lemma outside the namespace?

Reply:
Sure, I'll do that now.

Reply:
Ah, sorry, our messages crossed!  I'll `open`.  Should I also do similar replacements for the other `nolint`s?

Reply:
Yes, I think you should do this to all of them!

Reply:
(though it looks like for this one there's no need to `open`)